### PR TITLE
LGA-2305 create a docker development environment

### PR DESCRIPTION
**Pull request by MishNajam**
## What does this pull request do?

updated docker-compose file for local development and create run local script

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
       bash \
       py2-pip \
       tzdata \
       gettext
 
-RUN adduser -D app && \
-    cp /usr/share/zoneinfo/Europe/London /etc/localtime
-
+RUN adduser -D app
+RUN cp /usr/share/zoneinfo/Europe/London /etc/localtime
```
Comment (said-moj) [inline Dockerfile:13]: Is there a reason we split this into multiple run commands

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,30 +1,45 @@
...
+    depends_on:
+      - cla_backend
 
-db:
-  image: postgres
-  environment:
-    POSTGRES_USER: postgres
-    POSTGRES_PASSWORD: postgres
-    POSTGRES_DB: cla_backend
+  db:
```
[the remark this answers is not in the document]
Comment (jogilder) [reply docker-compose.yaml:12]: we need a writeable file system in this case.

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,30 +1,45 @@
-# NB: this is only used for *local development*
-# see Dockerfile for deployed version
+version: "3.9"
+services:
+  start_applications:
```
[the remark this answers is not in the document]
Comment (jogilder) [reply docker-compose.yaml:3]: we need a writeable file system in this case.

[the remark this answers is not in the document]
Comment (jogilder) [reply docker-compose.yaml:3]: we need a writeable file system in this case.

**Review by psweeting1 — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -134,32 +138,73 @@ Then `./manage.py translations push` to Transifex and `./manage.py translations
 
 See more detailed instructions in the [how-to guide](https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/1005060261/Produce+diagram+of+CLA+merits+decision+tree) on Confluence (log-in required).
 
+## Installation in a virtual environment
```
Comment (said-moj) [inline README.md:141]: @jogilder @MishNajam What do you think of moving this out to a separate documentation and linking to it from here?

**Review by psweeting1 — APPROVED** [review]
LGTM

**Review by said-moj — APPROVED** [review]
👍 LGTM

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**